docs/changelogs/v0.36.2.md

```
@@ -0,0 +1,18 @@
+
+## Security
+
+## Fixes
+
+## Dependencies
+
+## Development, Testing, and CI/CD
+
+---
+
+**Full Changelog**: https://github.com/cozystack/cozystack/compare/v0.36.1...main
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)